### DISCUSSION TITLE
Enderdragon drops

### DISCUSSION THREAD

**Discussion by Tizooo** [PurpurMC/Purpur#820] [__Dismissed__] [answered]
Possibilitie to let the enderdragon spawn a chest after he got killed after he got resummoned on the position where normally the dragon egg spawns and a config setting where u can define what sort of items should be in the chest. This would be extremely usefull for small and big server which have a pre loadet world and a border because elytras are then very limited and can't be found anymore this feature would add a cool way in getting elytras without making the border bigger so players can keep on finding elytras or other kind of items

**Comment by Tizooo** [comment]
I hope this is understandable I'm not the best in english ^^

**Comment by granny** [comment, marked as answer]
this sounds like a very specific use-case that’d be best done in a plugin. @YouHaveTrouble *might* add it to [PurpurExtras](https://github.com/YouHaveTrouble/PurpurExtras) when he has time but no promises.
